USAGE.md

```diff
@@ -8,9 +8,15 @@ Install rust: https://www.rust-lang.org/tools/install
 
 Install docker: https://docs.docker.com/desktop/
 
+### Setup environment variables
+
+Create a `signer.env` at the root of the file and add fields according to the [example file](./.env.example/signer.env.example). 
+
 ## Run the node
 
-Using docker you will be able to run all the required services locally.
+Using docker you will be able to run all the required services locally. </br>
+The `signer` service at the `SOCKET_PORT` specified in `signer.env` file.
+
```

Comment:
not needed. Docker doesn't read from this file. It used hardcoded values instead
```
  signer:
    image: "signer"
    restart: always
    environment:
      - ROOT_KEY=tprv8ZgxMBicQKsPeb6rodrmEXb1zRucvxYJgTKDhqQkZtbz8eY4Pf2EgbsT2swBXnnbDPQChQeFrFqHN72yFxzKfFAVsHdPeRWq2xqyUT2c4wH
      - SOCKET_PORT=10001

```